ScribusGeneratorBackend.py

```diff
@@ -401,7 +401,7 @@ def generate_templates(self, root, data: list) -> list:
                         version
                     )
 
-                    if index != min(records_in_document, data_count):
+                    if index > records_in_document:
```

Comment:
Thanks for the update.
This might require a test with a template that uses multiple data entries (i.e. a few %SG_NEXT-RECORD%, eg 5) and a data file that has some entries but LESS that needed to fill the whole document (eg 3).

I wanted to write a basic test suite against regression bugs for a long time... maybe now it would have been useful :/